SofascoreAcademy/SofascoreAcademy/Controllers/ViewController.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  ViewController.swift
+//  SofascoreAcademy
+//
+//  Created by Matija Pavicic on 11.03.2024..
+//
+
+import UIKit
+import SnapKit
+import SofaAcademic
+
+class ViewController: UIViewController {
+    let spainTournamentCell: TournamentCellView = .init()
+    let eventsStack: UIStackView = .init()
+    var eventsViewArray: [EventCellView] = .init()
+    var tournament = TournamentCellModel.sampleData
+    var events = EventCellModel.sampleData
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+        self.view.backgroundColor = .white
+        
+        self.view.addSubview(spainTournamentCell)
+        self.view.addSubview(eventsStack)
+       
+        eventsStack.axis = .vertical
+        
+        spainTournamentCell.countryLabel.text = tournament.countryName
+        spainTournamentCell.leagueLabel.text = tournament.leagueName
+        spainTournamentCell.logoImage.image = tournament.leagueLogo
+        
+        events.forEach {
+            let view: EventCellView = .init(matchStatus: $0.matchStatus)
+            view.homeTeamLogo.image = $0.homeTeamLogo
+            view.awayTeamLogo.image = $0.awayTeamLogo
+            view.homeTeamLabel.text = $0.homeTeamLabel
+            view.awayTeamLabel.text = $0.awayTeamLabel
+            view.homeScore.text = $0.homeTeamScore
+            view.awayScore.text = $0.awayTeamScore
+            view.startTime.text = EventCellController().convertMatchStartTimestamp(timestamp: $0.matchStartTime)
```

Comment:
Ovo bi trebao dobiti na kraju:
`view.startTime.text = EventDateHelper.getStartTime(for: $0.startTimeStamp)`

Slično za funkciju dolje